internal/weldr/api.go

```diff
@@ -1246,6 +1248,11 @@ func (api *API) composeHandler(writer http.ResponseWriter, request *http.Request
 
 	bp := api.store.GetBlueprintCommitted(cr.BlueprintName)
 
+	size := cr.Size
+	if size%MegaByte != 0 {
+		size = (size/MegaByte + 1) * MegaByte
```

Comment:
This needs some discussion / comment. The alternative is to refuse the compose if we don't like what is passed (we do that further down the stack). I think you are right that this is what we want, but would need some discussion as to why at this point?